hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/client/rpc/OzoneRpcClientTests.java

```diff
@@ -1092,18 +1092,18 @@ public void testDeleteAuditLog() throws Exception {
 
     // create a three replica file
     String keyName1 = "key1";
-    TestDataUtil.createKey(bucket, keyName1, THREE, RATIS, value);
+    TestDataUtil.createKey(bucket, keyName1, THREE, RATIS, value.getBytes(UTF_8));
```

Comment:
Let's change `String value` to `byte[] value` and store result of `getBytes(UTF_8)` once.  Then we can keep passing simply `value`.